### ISSUE TITLE
Load balancer for Kube-API Kubernetes in VPC 

### ISSUE THREAD

**Issue by leduyquy** [apache/cloudstack#9338]
##### ISSUE TYPE

 * Bug Report
 * Feature Idea

##### COMPONENT NAME

~~~
VPC, Kubernetes, Load Balancer 
~~~

##### CLOUDSTACK VERSION

~~~
 4.19.0.1 
~~~

##### CONFIGURATION

~~~
Create VPC 
Create Guest Network in VPC 
Create Kubernetes Cluster in Guest Network
~~~

##### OS / ENVIRONMENT

~~~
KVM and VMware host 
~~~

##### SUMMARY

~~~
Create Kubernetes in VPC. IP public access to KubeAPI use Forward rule not Load balancing
~~~

##### STEPS TO REPRODUCE

Without VPC
![Screenshot from 2024-06-21 16-47-47](https://github.com/apache/cloudstack/assets/68555139/3c6b93c3-e860-4c8f-ae62-0057311d47ad)

In VPC:
![vpc](https://github.com/apache/cloudstack/assets/68555139/d4c9d58a-5d3b-40f0-91ea-4d9c5602da4f)
~~~

~~~

##### EXPECTED RESULTS

~~~
 Load balancer for Kube-API running on VPC
~~~

##### ACTUAL RESULTS

~~~
Public IP forward to only one control node, when it down, Disconnect to cluster
~~~

**Comment by weizhouapache** [comment]
This issue is similar as #8295, which has been addressed by #7919 in main/4.20 branch

https://github.com/apache/cloudstack/blob/c6d3f6e623e35aadbe6e7d1db56d1d3244b55f09/plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/actionworkers/KubernetesClusterResourceModifierActionWorker.java#L762-L777

if we need this in 4.19, #8308 can be re-opened

**Comment by DaanHoogland** [comment]
@leduyquy can you check this on main branch, as @weizhouapache suggested?
thanks

**Comment by weizhouapache** [comment]
this duplicates #8295 , which has been fixed in main/4.20

@leduyquy 
can you please test the nightly build ?
https://download.cloudstack.org/testing/nightly/latest/
